modules/apps/headless-apio/forms/forms-apio-impl/src/main/java/com/liferay/forms/apio/internal/resource/FormInstanceNestedCollectionResource.java

```diff
@@ -73,7 +77,7 @@
 
 	@Override
 	public String getName() {
-		return "form-instance";
+		return "form-instances";
```

Comment:
I thought we were going to rename it to forms?

Reply:
I was thinking about it and I'm not sure if it's right to rename to forms and merge with the structure properties, it will become a huge resource. I prefer to keep it and discuss after finish the page's mapping. That's right for you?

Reply:
Ok!